packages/@aws-cdk/pipelines/README.md

```diff
@@ -544,6 +544,79 @@ const pipeline = new pipelines.CodePipeline(this, 'Pipeline', {
 });
 ```
 
+#### Deploying without change sets
+
+Deployment is done by default with `CodePipeline` engine using change sets, 
+i.e. to first create a change set and then execute it. This allows you to:
+
+- have approval before actual deployment happens
+- include [dynamic references](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html) 
```

Comment:
From the linked doc:

> When you use a dynamic reference, CloudFormation retrieves the value of the specified reference when necessary during _stack_ and change set operations.

Sort of sounds like normal stack operations also support them.  Was anyone able to try to see if it's a problem (or past experience)? I fully realize that the docs just might not reflect the reality of things.

Reply:
Oh gees, this added me as a reviewer, sorry. Please feel free to ignore.

Reply:
I think one example that can't be done with normal stack is that you have some value in secrets manager that you want to use at created resource, but resource itself doesn't support references. So e.g. with CodeBuild and ECS you can refer to secrets manager value with ARN, but if you make e.g. OIDC authentication at ALB level, you can only provide strings. With changeset it's possible to get the secret to ALB without having it in the template